keyboards/ergodone/keymaps/via/keymap.c

```diff
@@ -0,0 +1,216 @@
+ /* Copyright HarshitGoel96 2021
+  *  
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */ 
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#define QRTY 0 // qwerty above workman to make sure transparent does not catch unwanted keycode
+#define SYMB 1 // symbols
+#define MDIA 2 // media keys
+#define EXTRA 3 // added extra layer for via
+enum custom_keycodes {
+  PLACEHOLDER = SAFE_RANGE, // can always be here
+ 
+  VRSN,
+  
```

Comment:
The first custom keycode can, under the usual circumstances, always be set to `SAFE_RANGE`.

```suggestion
  VRSN = SAFE_RANGE, // can always be here
```